### DISCUSSION TITLE
Interaction with Kaspersky (and maybe other security softwares)

### DISCUSSION THREAD

**Discussion by ROdoka** [rustdesk/rustdesk#6331] [General]
### Bug Description

Hello!
In Kaspersky you have to allow remote softwares to be able to interact with its interface, else Kaspersky won't react to any clicks or keyboard actions you make on its popups etc.

I have allowed Rustdesk to interact but it still does not work as expected.
What makes me write this issue is: only one button of the Kaspersky popup/overlay window works: the [x] to close the window.
And I also can change the Kaspersky settings without being blocked by its security measures until this security window popup appears and wants you to confirm the changes.

### How to Reproduce

Remote connect to a (Windows) PC running Kaspersky and for example let it find the Rustdesk-1.2.3-x86_64.exe file from github.
It will (as always) nag you about a troublesome software and offers you to delete it or add it to an exceptions list.
If you want to add it to the list, a new window appears on screen where you have to confirm this action (so that it is not made by a malicious process).

The buttons in this window can not be clicked, except the [X] in the top right corner.
I'd like to add pictures but github says "something went really wrong, and we can't process that file", so no jpg or png worked.

### Expected Behavior

being able to click on the buttons

### Operating system(s) on local side and remote side

Win7 -> Win11 (also for Win10 -> Win11)

### RustDesk Version(s) on local side and remote side

1.2.4 -> 1.2.4

### Screenshots

I'd like to add pictures but github says "something went really wrong, and we can't process that file", so no jpg or png worked.

**Comment by ROdoka** [comment]
hmm. a discussion ...

It works for Win7 -> Win7, I just tested this.

**Comment by Living-Instinkt** [comment]
I have the same issue
